io.bit3.ThreemaQT.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.bit3.ThreemaQT
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: ThreemaQT
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
```

Comment:
Similarly to WhatsApp one, it should be `fallback-x11` if it works properly on Wayland.

Reply:
I will change it and try it out.